xarray/tests/test_dataset.py

```diff
@@ -4920,7 +4920,7 @@ def test_filter_by_attrs(self):
                 "temperature_10": (["t"], [0], temp10),
                 "precipitation": (["t"], [0], precip),
             },
-            coords={"time": (["t"], [0], dict(axis="T"))},
+            coords={"time": (["t"], [0], dict(axis="T", standard_name="time_in_seconds"))},
```

Comment:
```suggestion
            coords={"time": (["t"], [0], dict(axis="T", standard_name="time", units="s"))},
```

If using standard_name, I think it would be best to stay CF-compliant? Based on [the guidelines](http://cfconventions.org/Data/cf-standard-names/docs/guidelines.html) I don't think units should be given in the standard_name, but rather as its own attribute.

Reply:
I think this is a good idea, but I don't want to set the standard name to be the same as the name of the coordinate since it will make the code confusing to read. Perhaps i can change it to "time_after_1990" or something similar? Do you have a suggestion?

Reply:
I think if you want to use `standard_name`, it has to be "time" to be CF compliant. However, if you want "time_in_seconds" or "time_after_1990", I believe you can use the much-more lenient `long_name` attribute (see http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#long-name).

Reply:
Ahh, ok - I will switch to long_name then. Thanks for your help!